Framework/Beamline/src/DetectorInfo.cpp

```diff
@@ -86,7 +88,9 @@ bool DetectorInfo::isEquivalent(const DetectorInfo &other) const {
 
 /// Returns true if the detector with given detector index is a monitor.
 bool DetectorInfo::isMonitor(const size_t index) const {
-  // No check for time dependence since monitor flags are not time dependent.
```

Comment:
This comment seems to be still valid, but could be moved to L94. A similar one is preserved in L105. Could you re-instate it there?

Reply:
I put pack the comment